src/frequenz/repo/config/pytest/examples.py

```diff
@@ -163,6 +164,8 @@ def evaluate(self, example: Example) -> None | str:
         )
         # Remove first line (the line with the triple backticks)
         example_code = example_code[example_code.find("\n") + 1 :]
+        # Remove the last line (ending triple backticks)
+        example_code = example_code[: example_code.rfind("```") - 1]
```

Comment:
This is a bit of a naive approach, as markdown could also use other characters to delimit code blocks, for example:

```markdown
~~~py
code = "block"
~~~
```

But I guess it is OK as a quick fix. What I don't get is, if [sybil supports markdown](https://sybil.readthedocs.io/en/latest/markdown.html#markdown-parsers) why do we need any manual hack to get rid of the ending marker of the codeblock?

Reply:
BTW, I think we should use the markdown parser, not MyST, right? I think MkDocs uses the regular markdown parser.

Reply:
> What I don't get is, if [sybil supports markdown](https://sybil.readthedocs.io/en/latest/markdown.html#markdown-parsers) why do we need any manual hack to get rid of the ending marker of the codeblock?

because
```
        # Dedent the code example
        # There is also example.parsed that is already prepared, but it has
        # empty lines stripped and thus fucks up the line numbers.
```

see also https://github.com/simplistix/sybil/pull/117